GOVERNANCE.md

```diff
@@ -122,3 +122,10 @@ If an agenda item cannot reach a consensus, a CTC member can call for either a
 closing vote or a vote to table the issue to the next meeting. The call for a
 vote must be approved by a simple majority of the CTC or else the discussion
 will continue.
+
+A vote may be scheduled for a future date to allow time for CTC members to
+better educate themselves about relevant issues. A vote is not valid unless
+there is participation of a quorum of the CTC. A CTC quorum is more than half of
+the CTC. Absentee votes may be cast in advance by commenting on the weekly CTC
```

Comment:
> This sounds like the votes of more than half of those wo participate are needed for a simple majority in a vote (as opposed to more than half of the entire CTC). Is that correct?

This is incorrect.

It needs to be half of the _entire_ CTC.

See the above:

>  vote must be approved by a simple majority of the CTC

It does not say "of the CTC present", it says "**of the CTC**".